.python-version

```diff
@@ -1 +1 @@
-3.6.12
+3.9
```

Comment:
Dependabot was using this decide which version of Python to use for installing dependencies which prevented it from working correctly since Python 3.6 is EOL. I also updated the vercel runtime in our deployment so we should be fine with using a newer Python version here 🤞